src/controller/CHIPDeviceController.cpp

```diff
@@ -929,6 +933,18 @@ void DeviceCommissioner::OnSessionEstablished()
     }
 
     ChipLogDetail(Controller, "Remote device completed SPAKE2+ handshake\n");
+
+    if (mOperationalCredentialsDelegate != nullptr)
```

Comment:
should there be a log on the else branch here?
or is it acceptable to not have an opcert delegate in which case we are not trying to ask for a certificate signing?

I find this a bit harder to follow as we check for a nullptr but inside of the if block we never use said pointer. I am more used to `if (foo != nullptr) { ... use foo here ... }`

Consider pulling this logic into the 'Send' method.

Reply:
This code block is actually just a place holder, as we haven't integrated it to the cluster API. I do see that it's very confusing as it is right now. I'll update it, and add a TODO.